Description: char is unsigned on riscv64, so build the uint8 version of uname
diff --git a/src/go/plugins/system/uname/uname_uint8.go b/src/go/plugins/system/uname/uname_uint8.go
index 72d3c36..4f6ce07 100644
--- a/src/go/plugins/system/uname/uname_uint8.go
+++ b/src/go/plugins/system/uname/uname_uint8.go
@@ -1,4 +1,4 @@
-// +build linux,arm linux,ppc64le linux,s390x
+// +build linux,arm linux,ppc64le linux,s390x linux,riscv64
 
 /*
 ** Zabbix
